core/src/smartcontracts/wasm.rs

```diff
@@ -867,7 +927,7 @@ mod tests {
 
     #[test]
     fn execute_query_exported() -> Result<(), Error> {
-        let account_id = AccountId::from_str("alice@wonderland")?;
+        let account_id = AccountId::from_str("alice@wonderland").expect("Valid");
```

Comment:
Relevant comment from the other PR: [link](https://github.com/hyperledger/iroha/pull/3106#discussion_r1121635117).